nsqadmin/static/js/views/channel.js

```diff
@@ -21,11 +21,18 @@ var ChannelView = BaseView.extend({
     initialize: function() {
         BaseView.prototype.initialize.apply(this, arguments);
         this.listenTo(AppState, 'change:graph_interval', this.render);
-        this.model.fetch().done(function() {
-            this.template = require('./channel.hbs');
-            this.render();
-            Pubsub.trigger('view:ready');
-        }.bind(this));
+        this.model.fetch()
+            .done(function(data) {
+                    this.template = require('./channel.hbs');
+                    this.render({'message': data['message']});
+                }.bind(this))
+            .fail(function(jqXHR) {
+                    this.template = require('./error.hbs');
+                    this.handleViewError(jqXHR);
+                }.bind(this))
+            .always(function() {
+                    Pubsub.trigger('view:ready');
+                });
```

Comment:
Muchisimos tabs, Batman. Personally, I'd unindent the bodies of these functions by one tab. But that's just a style comment. Take it or leave it :grimacing: